package.json

```diff
@@ -1,25 +1,32 @@
 {
   "name": "@novnc/novnc",
-  "version": "1.6.0",
+  "version": "1.6.1-alpha.1",
   "description": "An HTML5 VNC client",
-  "browser": "lib/rfb",
-  "directories": {
-    "lib": "lib",
-    "doc": "docs",
-    "test": "tests"
-  },
```

Comment:
Are these not used anymore?

Reply:
See npm js docs: [directories](https://docs.npmjs.com/cli/v11/configuring-npm/package-json#directories)

> The CommonJS [Packages](http://wiki.commonjs.org/wiki/Packages/1.0) spec details a few ways that you can indicate the structure of your package using a directories object. If you look at [npm's package.json](https://registry.npmjs.org/npm/latest), you'll see that it has directories for doc, lib, and man.
> 
> In the future, this information may be used in other creative ways.

Since this is no longer exported using ComonJS Modules, I am not sure who would use this.